pkg/splunk/enterprise/clustermaster.go

```diff
@@ -164,6 +164,9 @@ func validateClusterMasterSpec(cr *enterprisev1.ClusterMaster) error {
 // getClusterMasterStatefulSet returns a Kubernetes StatefulSet object for a Splunk Enterprise license master.
 func getClusterMasterStatefulSet(client splcommon.ControllerClient, cr *enterprisev1.ClusterMaster) (*appsv1.StatefulSet, error) {
 	var extraEnvVar []corev1.EnvVar
+	for _, envVar := range cr.Spec.CommonSplunkSpec.ExtraEnv {
```

Comment:
Since the extraEnv parameter is a part of `CommonSplunkSpec`(which is available to use in the function `getSplunkStatefulSet`), could we add the logic to adding to the env variables [here](https://github.com/splunk/splunk-operator/blob/develop/pkg/splunk/enterprise/configuration.go#L652) in the following manner?

```
for _, envVar := range spec.CommonSplunkSpec.ExtraEnv {
    	extraEnvVar = append(env, envVar)
}
```

This would involve making a change in a single place in the repo vs CR specific changes.

Just a suggestion, the changes look good otherwise.

Reply:
Ya good point.  I had put the code in the `extraEnv` param for `getSplunkStatefulSet()` since it already had the extraEnv variable that was NULL'ed out for most CRDs.  Though since its common it makes a lot more sense to put it in the common area for all types.  I'll adjust the diff to reflect this.

Reply:
Looks great. Thanks Jeff